lib/ClangImporter/SwiftDeclSynthesizer.cpp

```diff
@@ -3183,7 +3183,7 @@ FuncDecl *SwiftDeclSynthesizer::makeAvailabilityDomainPredicate(
       BuiltinIntegerType::get(1, ctx), ImporterImpl.ImportedHeaderUnit);
   funcDecl->setBodySynthesizer(synthesizeAvailabilityDomainPredicateBody,
                                (void *)var);
-  funcDecl->setAccess(AccessLevel::Internal);
+  funcDecl->setAccess(AccessLevel::Private);
```

Comment:
Why `private` rather than marking them as `@export(implementation)`, which allows the linker to merge the different copies in different translation units?

Reply:
Do we worry about deserialization issues trigged by the function body of this synthesized function when marked as `@export(implementation)`? Previously, we have been hitting a module deserialization error (e.g.rdar://164410957).